src/constants.ts

```diff
@@ -1,10 +1,10 @@
 export const SocialLinks = {
   GitHub: 'https://github.com/MinaProtocol/mina',
   Twitter: 'https://twitter.com/minaprotocol',
-  WeChat: 'https://forums.minaprotocol.com/',
+  WeChat: 'https://minafoundation.notion.site/Follow-Mina-s-WeChat-and-Weibo-Official-Accounts-91226f4ea59b402ebf0cb3b40f764d69',
   YouTube: 'https://www.youtube.com/c/minaprotocol',
-  Discord: 'https://discord.gg/minaprotocol',
-  Telegram: 'http://bit.ly/MinaTelegram',
+  Discord: 'https://bit.ly/MinaDiscord',
```

Comment:
Oh no actually, at the time I tested the original link wasn't working properly, but I just noticed it was due to my VPN. Would revert the changes. (Otherwise, this is what MF currently uses btw).

Reply:
we'll use the bitly URL shortened link that MF uses to track